static/app/components/organizations/pageFilters/container.tsx

```diff
@@ -34,6 +34,10 @@ type Props = InitializeUrlStateProps & {
    * Custom alert message for the desynced filter state.
    */
   desyncedAlertMessage?: string;
+  /**
+   * Whether to hide the desynced filter alert.
+   */
+  hideDesyncAlert?: boolean;
```

Comment:
In the future, <DesyncedFilterAlert /> will be completely replaced by the new banner. Until then, we'll need to use this prop to conditionally hide it as we migrate pages to the new page filter components.